Tests/modules/network_related/test__socket.py

```diff
@@ -180,6 +180,11 @@
                  "TCP_SYNCNT" : 7,
                  "TCP_WINDOW_CLAMP" : 10}
 
+def find_free_port():
+    s = socket.socket()
+    s.bind(('', 0))
+    return s.getsockname()[1]
```

Comment:
Indeed, that appears to solve the issue.

Reply:
It still fails on my Linux system. Since it is reproducible, I think I will be able to investigate it. Also `test_function` fails. If I find the solution, I will open another PR.